CMakeLists.txt

```diff
@@ -53,7 +53,10 @@ endif()
 
 option(SENTRY_ENABLE_INSTALL "Enable sentry installation" "${SENTRY_MAIN_PROJECT}")
 
-if(APPLE OR WIN32)
+if(MSVC AND CMAKE_GENERATOR_TOOLSET MATCHES "_xp$")
+	message(WARNING "Backend is set to 'none' for MSVC with XP toolset")
```

Comment:
hm, I was thinking if we should remove this warning. But since crashpad in the only supported backend on windows right now, it is good to be more vocal about this, to not raise false expectations for users.

Reply:
changed wording a bit to be a more concrete.